spec/API_specification/creation_functions.md

```diff
@@ -289,6 +289,33 @@ Returns evenly spaced numbers over a specified interval.
 
     -   a one-dimensional array containing evenly spaced values.
 
+(function-meshgrid)=
+### meshgrid(*arrays, /, *, indexing='xy')
+
+Returns coordinate matrices from coordinate vectors.
+
+### Special cases
+
+-    For `n` one dimensional arrays `x1, x2, ..., xn` with lengths `Ni = len(xi)`, this function returns `(N1, N2, N3, ..., Nn)` shaped arrays if the indexing is 'ij' or `(N2, N1, N3, ..., Nn)` shaped arrays if indexing 'xy'.
+
+-    The 0-D and 1-D case, the indexing keyword has no effect.
+
+### Parameters
+
+-    ***arrays**: _&lt;array&gt;_
```

Comment:
This should be `**\*arrays**` to ensure it doesn't get rendered with a space between `*` and `arrays`.

Reply:
I think we can omit the third `*` altogether. We haven't included it elsewhere (e.g., `result_type`, etc).